### DISCUSSION TITLE
[feature request] make a suggestion. Can you add a carriage return event to each item in the El form

### DISCUSSION THREAD

**Discussion by yunziyan11** [element-plus/element-plus#6759] [Ideas]
### Existing Component
否

### Component Name
无组件

### Description
只是一个建议  比如form表单里给每一个字段添加回车事件  自动校验，校验失败则提示失败，焦点留在当前字段上， 如果校验成功则跳转到下一个字段获取焦点，这样的话 只要第一个元素获取焦点 后面的元素只要输入或者选择完毕，只要回车就能跳转到下一个元素获取焦点 就不需要鼠标来回点击了 还能快速的操作， 能节省大量的时间成本

[a comment by an automated account is not shown]

**Comment by XingXiaoWu** [comment]
如果校验成功则跳转到下一个字段获取焦点，校验不成功就不跳了么。。。这不合适吧。
只是获取焦点的话，为什么不用tab键

**Comment by yunziyan11** [comment]
当然这只是我的一个想法&nbsp; 对于用户而言，就拿我来说吧 我的习惯就是按回车切换到下一个或者下一行(这很容易实现 指令或者其他的方法都可以)，&nbsp; 当然也可以按tab键 对于有些用户他只知道enter能切换(下意识的就会去按 enter)&nbsp; 很少用到tab。
对于校验不成功不跳转 我的想法是 一个字段既然是需要满足某种规则的，如果没有满足某种规则就跳转的话，最终提交还是要回来满足规则才能提交表单 那就不如直接在切换焦点的时候不跳转 并且给与错误提示，满足后再跳转。




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "element-plus/element-plus"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年9月17日(星期五) 上午10:25
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [element-plus/element-plus] [feature request] make a suggestion. Can you add a carriage return event to each item in the El form (#3451)





 
如果校验成功则跳转到下一个字段获取焦点，校验不成功就不跳了么。。。这不合适吧。
 只是获取焦点的话，为什么不用tab键
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by XingXiaoWu** [comment]
个人觉得觉得不合理，且没必要

**Comment by sxzz** [comment]
个人建议自行实现。
